lib/postprocess/image.js

```diff
@@ -93,6 +95,19 @@ module.exports = async files => {
           response.headers['content-type'] || response.headers['Content-Type']
         imageExtensions = contentType.split('/')[1]
         imageBuffer = Buffer.from(response.data)
+
+        // check if the file is valid
+        try {
+          sizeOf(imageBuffer)

```

Comment:
Hm interesting. A cheap way would be to check the buffer length. I would check the internals of `sizeOf` as it might even check file headers etc. which wouldn’t be a bad thing afterall.

Reply:
It also tries to infer the type of the buffer and throws an error if the filetype is not supported (since it only works with images), which might be useful as well - this should also help in case the buffer is not empty but rather damaged/invalid in some other way -> I tested it with a broken base64 string and the buffer obviously has a length but throws a type error, which I think is useful